ui/admin/tests/e2e/tests/delete-resources.spec.js

```diff
@@ -0,0 +1,197 @@
+  createNewControllerLedWorkerCli,
+  createNewPasswordAuthMethodCli,
+  makeAuthMethodPrimaryCli,
+  createNewPasswordAccountCli,
+  createNewRoleCli,
+  createNewGroupCli,
+  createNewUserCli,
+  createNewStaticHostCatalogCli,
+  createDynamicAwsHostCatalogCli,
+  createNewStaticHostCli,
+  createNewHostSetCli,
+  createNewStaticCredentialStoreCli,
+  createNewTcpTarget,
+  createNewSshTarget,
+  createNewVaultCredentialStoreCli,
+  createNewUsernamePasswordCredentialCli,
+  deleteOrgCli,
+} = require('../helpers/boundary-cli');
+
+const {
+  deleteResource,
+  removeAuthMethodAsPrimary,
+} = require('../helpers/boundary-ui');
+const { checkVaultCli } = require('../helpers/vault-cli');
+const { execSync } = require('child_process');
+
+const secretPolicyName = 'kv-policy';
+const boundaryPolicyName = 'boundary-controller';
+
+test.use({ storageState: authenticatedState });
+
+test.beforeAll(async () => {
+  await checkEnv([
+    'VAULT_ADDR',
+    'VAULT_TOKEN',
+    'E2E_VAULT_ADDR',
+    'E2E_TARGET_ADDRESS',
+    'E2E_SSH_USER',
+    'E2E_SSH_KEY_PATH',
+    'E2E_AWS_ACCESS_KEY_ID',
+    'E2E_AWS_SECRET_ACCESS_KEY',
+  ]);
+  await checkBoundaryCli();
+  await checkVaultCli();
+});
+
+test.beforeEach(async () => {
+  execSync(`vault policy delete ${secretPolicyName}`);
+  execSync(`vault policy delete ${boundaryPolicyName}`);
+  await authenticateBoundaryCli();
+});
+
+test('Verify resources can be deleted', async ({ page }) => {
+  let orgId;
+  try {
+    // Create boundary resources using CLI
+    orgId = await createNewOrgCli();
+    let projectId = await createNewProjectCli(orgId);
+    let workerId = await createNewControllerLedWorkerCli();
+    let authMethodId = await createNewPasswordAuthMethodCli(orgId);
+    await makeAuthMethodPrimaryCli(orgId, authMethodId);
+    let passwordAccountId = await createNewPasswordAccountCli(authMethodId);
+    let projectScopeRoleId = await createNewRoleCli(projectId);
+    let orgScopeRoleId = await createNewRoleCli(orgId);
+    let globalScopeRoleId = await createNewRoleCli('global');
+    let groupId = await createNewGroupCli(orgId);
+    let userId = await createNewUserCli(orgId);
+    let staticHostCatalogId = await createNewStaticHostCatalogCli(projectId);
+    let dynamicAwsHostCatalogId = await createDynamicAwsHostCatalogCli(
+      projectId
+    );
+    let staticHostId = await createNewStaticHostCli(staticHostCatalogId);
+    let staticHostSetId = await createNewHostSetCli(staticHostCatalogId);
+    let staticCredentialStoreId = await createNewStaticCredentialStoreCli(
+      projectId
+    );
+    let vaultCredentialStoreId = await createNewVaultCredentialStoreCli(
+      projectId,
+      secretPolicyName,
+      boundaryPolicyName
+    );
+    let usernamePasswordCredentialId =
+      await createNewUsernamePasswordCredentialCli(staticCredentialStoreId);
+    let tcpTargetId = await createNewTcpTarget(projectId);
+    let sshTargetId = await createNewSshTarget(projectId);
```

Comment:
How do you suggest to organize this? Create a `delete-resources-ent.spec.js` file with enterprise related resourses covered?

Reply:
Similar to the BE test suite. I think it will need to live in `boundary-ui-enterprise`. I don't know if the FE team has a specific naming convention for enterprise-only files.

Reply:
@calcaide @ZedLi Do you agree with this approach? If so, are there naming conventions for enterprise-only files?

Reply:
Oh... I did not know there were no enterprise-only files... so how is `boundary-ui-enterprise` used? 

If we're keeping it all in this repo, then maybe one option is to use the tag feature: https://playwright.dev/docs/test-annotations#tag-tests

Reply:
Might be good to sync on this but I don't think it needs to live in the `boundary-ui-enterprise` repo. We haven't had any "enterprise repo" specific files besides github workflows yet.

Our approach to the repos are also different from the backend. We actually have all of the enterprise features in `boundary-ui` gated behind feature flags. The frontend is somewhat different since it doesn't have all the features kept in separate repos as the true gatekeeping happens from access to the boundary enterprise binary.

Reply:
@moduli 
Long story short. The UI is baked with all the features and then we use flags to determinate what we "turn" OFF or ON. In fact, in `boundary-ui` we define features specific to OSS, Enterprise and HCPb (is not the cloud-ui you see on the portal but the UI that runs the binary deployed within HCP). [Here is the features file](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/config/features.js) as example.

As @ZedLi mentioned, I think would be good to do a quick sync about this.